staking/apr/compute.go

```diff
@@ -23,7 +22,7 @@ type Reader interface {
 }
 
 const (
-	secondsInYear = time.Second * 3.154e+7
+	secondsInYear = int64(3.154e+7)
```

Comment:
not sure why the google top result gives the wrong answer, but 31557600 
 is a better estimate? https://www.rapidtables.com/calc/time/seconds-in-year.html

Reply:
Sure will address in next pr